drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties

```diff
@@ -13,13 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-newScenarioSimulationDescription=Test Scenario (Preview)
+newScenarioSimulationDescription=Test Scenario 
```

Comment:
There is a redundant space after the name.

> Test Scenario_

Reply:
fixed